recipes/tagine.md

```diff
@@ -0,0 +1,58 @@
+# Lubes' Vegetable Tagine
```

Comment:
In the README you called it Vegetable Tagine. I dont think you need your name there :grin: